### PR TITLE
Add full screen (presentation) mode

### PR DESCRIPTION
**Pull request by aziz512**
Add a shortcut that switches the window into full screen mode.

### CODE REVIEW

**Comment by kirjs** [comment]
Maybe add a button?

**Comment by AleksandrBu** [comment]
If I understand it right, it's the same as f11 pressing does?

**Comment by AleksandrBu** [comment]
> Maybe add a button?

Please, tell me what exactly we wanna get?

**Comment by NothingEverHappens** [comment]
Like a full screen round button in the corner

**Comment by NothingEverHappens** [comment]
Also maybe use alt+enter or ctrl+enter?

File: apps/codelab/src/app/components/buttons-nav-bar/menu-fullscreen-widget/menu-fullscreen-widget.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+@Component({
+  selector: 'codelab-menu-fullscreen-widget',
+  templateUrl: './menu-fullscreen-widget.component.html',
+  styleUrls: ['./menu-fullscreen-widget.component.scss']
+})
+export class MenuFullscreenWidgetComponent implements OnInit {
+
+  constructor() { }
+
+  handleClick() {
```
Comment (kirjs) [inline apps/codelab/src/app/components/buttons-nav-bar/menu-fullscreen-widget/menu-fullscreen-widget.component.ts]: Call this smth like OpenFullScreen()
and do   document.documentElement.requestFullscreen(); directly here

**Pull request merged**